src/components/DistanceEReceipt.js

```diff
@@ -91,7 +91,8 @@ function DistanceEReceipt({transaction}) {
                                     key={key}
                                 >
                                     <Text style={styles.eReceiptWaypointTitle}>{translate(descriptionKey)}</Text>
-                                    <Text style={styles.eReceiptWaypointAddress}>{waypoint.address || ''}</Text>
+                                    <Text style={styles.eReceiptWaypointAddress}>{waypoint.name || ''}</Text>
```

Comment:
Again rather than rendering empty text let's avoid rendering the component.

Can we please make the name of the style match or make it more generic?

If we are going to fallback on the address when there's no name for the waypoint editor, we should also do it here. However now I'm starting to think I disagree with @dubielzyk-expensify's suggestion.

The different styles for the name and address text make a clear distinction about what each represents. If the name is missing I don't see why we should display the address with the same style, since it has a different meaning. It also makes the code a bit ugly.

Reply:
I have made handled it the same way we handle predefined places in `renderRow` by setting the title to the address if name is not defined. Let me know if that looks good